<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>112</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>112</height>
       </size>
      </property>
      <property name="title">
       <string>Формат</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Шрифт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_Font">
           <property name="text">
            <string>Настроить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Цвет текста</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_TextColor">
           <property name="text">
            <string>Настроить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Цвет фона</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_BackColor">
           <property name="text">
            <string>Настроить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Стиль заливки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_BackStyle">
           <item>
            <property name="text">
             <string>Solid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dense1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dense2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dense3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dense4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dense5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dense6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dense7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No Brush</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Кегль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Size">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>72</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Выравнивание</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_Align">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>Слева</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По центру</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Справа</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Bold">
           <property name="text">
            <string>Полужирный</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Italic">
           <property name="text">
            <string>Курсив</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Underline">
           <property name="text">
            <string>Подчеркнутый</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Table">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Расписание преподавателя</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>128</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_packet"/>
    <addaction name="action_prepod"/>
    <addaction name="separator"/>
    <addaction name="action_help"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_packet">
   <property name="text">
    <string>Пакетная обработка</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_prepod">
   <property name="text">
    <string>Инфо о преподавателях</string>
   </property>
  </action>
  <action name="action_table">
   <property name="text">
    <string>Расписание преподавателя</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
